FordLabs/PeopleMover
# My Roles Call-To-Action

## Issue
Resolves #425 

## What was done
- [x] Added edit and delete icons
- [x] Changed styling to match figma design

## How to test
1. Go to my roles modal and see updated edit and delete features for roles

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
